metrics/baseline/schemas/baseline.sql

```diff
@@ -0,0 +1,79 @@
+CREATE EXTERNAL TABLE IF NOT EXISTS foxsec_metrics/baseline (
```

Comment:
Because although it was initially generated it did still need some minor changes. 
Athena table defns also support loads of useful features like serdeproperties which we would need to specify manually. 
And we wont be able to generate tables for csv / tsv formats.
So we should save all table defns, even those generated, for future reference.
And we should maintain them so they never become invalid - if they are no longer used then they should be deleted.
Happy to have comments to say how they were created.

Reply:
Okay - I missed the obvious -- with Athena, there is no way to backup/restore the schemas. So if someone misclicks, bye bye table. We need the table create statements kept to recover from that.